source/dpp/translation/macro_.d

```diff
@@ -366,8 +366,11 @@ private auto fixCasts(R)(
 
         if( // const type
             tokens.length >= 2
-            && tokens[0] == Token(Token.Kind.Keyword, "const")
-            && isType(tokens[1..$])
+            && ((tokens[0] == Token(Token.Kind.Keyword, "const")
+                    && isType(tokens[1..$]))
+                || (tokens[$-1] == Token(Token.Kind.Keyword, "const")
```

Comment:
This makes sure that we treat all the cases with const + pointers, and view them as casts.

However, DPP does not currently translate the C cast within a macro. So that means that "(struct foo const *)" or "(struct foo * const)" are not going to be translated to valid D types (although now they are at least going to be seen as casts)